packages/manager/apps/vrack-services/src/utils/cart.ts

```diff
@@ -0,0 +1,43 @@
+import { CartItem, createCart } from '@ovh-ux/manager-module-order';
+
+const vrackItem: CartItem = {
+  itemEndpoint: 'vrack',
+  options: {
+    duration: 'P1M',
+    planCode: 'vrack',
+    pricingMode: 'default',
+    quantity: 1,
+  },
+};
+
+export const createVrackServicesCart = async ({
+  ovhSubsidiary,
+  region,
+  hasVrack,
+}: {
+  ovhSubsidiary: string;
+  region?: string;
+  hasVrack?: boolean;
+}) =>
+  createCart({
+    ovhSubsidiary,
+    items: [
+      region && {
+        itemEndpoint: 'vrackServices',
+        options: {
+          duration: 'P1M',
+          planCode: 'vrack-services',
+          pricingMode: 'default',
+          quantity: 1,
+        },
+        configurations: [{ label: 'region', value: region }],
+      },
+      hasVrack && vrackItem,
+    ].filter(Boolean),
```

Comment:
Why is filter needed ?

Reply:
If I don't filter it will keep a false (in case there is no region or no vrack) and do a call that will fail